game/views.py

```
@@ -576,7 +619,126 @@ def verify_otp(request):
+        return JsonResponse({
+            'success': True,
+            'message': 'OTP resent successfully.',
+            'cooldown': OTP_COOLDOWN_SECONDS,
+        })
+
+    except (SMTPException, BadHeaderError, OSError):
+        return JsonResponse(
+            {'success': False, 'message': 'Failed to send OTP email. Please try again.'},
+            status=500,
+        )
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)